app/routes.py

```diff
@@ -1,2 +1,251 @@
-from flask import Blueprint
+from app import db
+from flask import Blueprint, request, make_response, jsonify, abort
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime, timedelta, date
+import requests
+import os
 
+
+#blueprints
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+
+#helper functions
+def valid_customer_input(form_data):
+    if "name" not in form_data:
+        abort(make_response({"error": "Missing customer name"},400)) 
+    if "postal_code" not in form_data:
+        abort(make_response({"error": "Missing customer postal_code"},400))
+    if "phone" not in form_data:
+        abort(make_response({"error": "Missing customer phone"},400))
+    return True
+
+
+def valid_video_input(form_data):
+    if "title" not in form_data:
+        abort(make_response({"error": "Missing video title"},400))
+    if "release_date" not in form_data:
+        abort(make_response({"error": "Missing video release_date"},400))
+    if "total_inventory" not in form_data:
+        abort(make_response({"error": "Missing video total_inventory"},400))
+    return True
+
+#routes
+@customers_bp.route("", methods=["GET"])
+def get_customers():
+    customers = Customer.query.all()
+    customers_response = [customer.to_dict() for customer in customers]
+    return jsonify(customers_response)
+
+
+@customers_bp.route("/<customer_id>", methods=["GET"])
+def get_one_customer(customer_id):
+    customer = Customer.query.get(customer_id)
+    if not customer:
+        return ({"error": f"No customer with ID #{customer_id}"}, 404)
```

Comment:
You can simplify this using `get_or_404`:

```suggestion
    customer = Customer.query.get_or_404(customer_id)
```